pom.xml

```diff
@@ -195,6 +195,9 @@
           <failOnViolation>true</failOnViolation>
           <failsOnError>true</failsOnError>
           <linkXRef>false</linkXRef>
+          <sourceDirectories>
```

Comment:
I see in the [open issue on this](https://github.com/dapr/java-sdk/issues/972), you mentioned that there are in fact checkstyle violations. `[ERROR] Failed to execute goal org.apache.maven.plugins:maven-checkstyle-plugin:3.1.1:check` locally. I do not see those same failures/warnings showing up in CI now. Maybe we should add that command you ran locally to CI and only merge once there are no `checkstyle` violations.